Sources/Sentry/Public/SentryTransactionSamplingContext.h

```diff
@@ -0,0 +1,33 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class SentryTransactionContext;
+
+NS_SWIFT_NAME(TransactionSamplingContext)
+@interface SentryTransactionSamplingContext : NSObject
```

Comment:
`h`: In Java this is called `SamplingContext`. Did we agree on another name, @marandaneto ?
```suggestion
NS_SWIFT_NAME(SamplingContext)
@interface SentrySamplingContext : NSObject
```

Reply:
For this PR I used .NET as reference, maybe that's the reason of the discrepancies.